packages/zoe/src/contractSupport/ratio.js

```diff
@@ -352,6 +352,10 @@ export const ratiosSame = (left, right) => {
 export const quantize = (ratio, newDen) => {
   const oldDen = ratio.denominator.value;
   const oldNum = ratio.numerator.value;
+  if (oldDen < newDen) {
+    return ratio;
+  }
+
```

Comment:
this is a change in behavior and I think breaks the declared behavior,
> Make an equivalant ratio *with a new denominator*

This function `quantize` quantizes. Deciding not to quantize is the responsibility of the caller.